cli/install/template.go

```diff
@@ -981,8 +981,6 @@ data:
     env:
     - name: LINKERD2_PROXY_LOG
       value: warn,linkerd2_proxy=info
-    - name: LINKERD2_PROXY_BIND_TIMEOUT
-      value: {{.ProxyBindTimeout}}
```

Comment:
This change also needs to be reflected in the test fixtures for the proxy-injector package.

```
./controller/proxy-injector/fake/data/config-proxy.yaml:4:- name: LINKERD2_PROXY_BIND_TIMEOUT
./controller/proxy-injector/fake/data/inject-sidecar-container-spec.yaml:4:- name: LINKERD2_PROXY_BIND_TIMEOUT
```

And updating those files will require re-encoding the patches in these files:

```
controller/proxy-injector/fake/data/inject-no-labels-response.yaml
controller/proxy-injector/fake/data/inject-enabled-response.yaml
```

You can do that by copying the test output and passing it through `base64`. Apologies this isn't more straightforward -- we'll improve upon it as part of #1748.

Reply:
Thanks, I think c569f65b58af23e7e7a61f6b34831cdf1c346b77 should address this?